cloudbuild-staging.yaml

```diff
@@ -1,11 +1,9 @@
 steps:
-- name: 'node:10'
+- name: 'node:12'
   entrypoint: npm
   args: ['ci']
-- name: 'node:10'
+- name: 'node:12'
   entrypoint: npm
-  args: ['run', 'build-ci']
-  env:
-    - 'PATH_PREFIX=staging.nodejs.dev/$SHORT_SHA'
+  args: ['run', 'build']
 - name: 'gcr.io/cloud-builders/gsutil'
-  args: ['-m', 'rsync', '-R', 'public', 'gs://staging.nodejs.dev/$SHORT_SHA']
+  args: ['-m', 'rsync', '-R', 'public', 'gs://staging.nodejs.dev/']
```

Comment:
If this is deploying to the root of staging, is that going to conflict/overwrite those directories?

Reply:
It shouldn't be. the directories for staging are numbers generated from PR number. AFAIK there is no directory generated by the website that is just an integer.

If we wanted to be safe we could move the generation of preview to either another subdomain or a folder down such as `staging.nodejs.dev/preview`

We would just have to make sure to not reuse that folder for the website.